frappe/utils/__init__.py

```diff
@@ -277,7 +277,7 @@ def get_traceback(with_context=False) -> str:
 
 
 def log(event, details):
-	frappe.logger().info(details)
+	frappe.logger(event).info(details)
```

Comment:
Is this even used anywhere? 🤔

Reply:
rest LGTM.

Reply:
On none of the apps on my bench (I have lots); should probs just get rid of it.

Reply:
But don't want to break something like this 🤔

Reply:
oh the Irony HAHA